docker/start.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+count=0
```

Comment:
- check indentation ? (separate PR is ok)
- this should be in the base image, generated by the proxy package, so it can be reused for 'mesh expansion'.

Most files should be moved there - it is also ok to do that in a separate PR, to unblock the testing, but it must be done.